### PR TITLE
Integrating manage booking

### LINKED ISSUE #31: IN: Manage bookings
Blocked by: Manage bookings UI

Integrate management of bookings with requests so that it has the ability to 
1. accept/decline a booking

To test incoming requests, by this point you should have created controllers for requests, you can create multiple requests to make sure it's integrated well

### PR DESCRIPTION
**Pull request by rizperdana**
### What this PR does (required):
- This MR will do integration between backend and frontend of manage booking

### Any issues with the current functionality (optional):
- Still fail to do manage booking

### CODE REVIEW

**Comment by sundayezeilo** [comment]
Hey @rizperdana, kindly link your ticket.

File: server/routes/request.js [hunk 1]
```diff
@@ -1,17 +1,14 @@
...
 } = require("../controllers/request");
 
-router.route("/").get(protect, loadRequests);
-
-router.route("/:id").post(protect, newRequest);
-
-router.route("/:requestId").patch(protect, updateRequest);
+router.route("/").get(protect, getRequests);
+router.route("/").post(protect, createRequest);
+router.route("/:requestId").patch(protect, updateRequestStatus);
```
Comment (sundayezeilo) [inline server/routes/request.js:12]: :+1:

**Review by sundayezeilo — COMMENTED** [review]
Good work with this one.

**Comment by rizperdana** [comment]
> Hey @rizperdana, kindly link your ticket.

@ezeilo-su forget that linked issue was on side bar, thank you for reminder

**Review by moffatethan — APPROVED** [review]

**Pull request merged**